### DISCUSSION TITLE
小花火

### DISCUSSION THREAD

**Discussion by Mcayear** [share121/inter-knot#4703] [崩坏：星穹铁道]
![Image_1720591849510](https://github.com/user-attachments/assets/4bd05a24-a09d-4729-b6db-6c36b17a6321)

欢迎来玩，基岩版 Minecraft 独特的 RPG 服
IP：tale.0mc.me
端口：11013

**Comment by lanximuli** [comment]
米斯卡，莫斯卡，小花火~~~！

**Comment by lllliufeng** [comment]
我是jvav玩家（

**Reply by nodrymeanwet** [reply]
那是 java ( ˘•ω•˘ )

**Reply by lllliufeng** [reply]
> 那是 java ( ˘•ω•˘ )

是一个以前的老梗啦，[看这](https://blog.csdn.net/qq_45048713/article/details/105888603)
